elements/pl-file-editor/pl-file-editor.js

```diff
@@ -77,9 +83,25 @@ window.PLFileEditor = function (uuid, options) {
   }
   this.setEditorContents(currentContents);
 
+  this.syncSettings();
+
+  this.initSettingsButton(uuid);
+
   this.initRestoreOriginalButton();
 };
 
+window.PLFileEditor.prototype.syncSettings = function () {
+  window.addEventListener('storage', (event) => {
+    if (event.key === 'pl-file-editor-theme') {
+      this.editor.setTheme(event.newValue);
+    }
+  });
+
+  window.addEventListener('storageSync', () => {
```

Comment:
Let's namespace this event to this specific element, i.e. name this something like `pl-file-editor-settings-changed`.